<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QGORIKD" name="Test" tests="7" failures="1" timestamp="17 Sep 2019 16:45:22 GMT" time="0.096" errors="0">
  <testcase name="Demo" time="0.022" classname="Test.Day1"/>
  <testcase name="Demo_2" time="0.005" classname="Test.Day1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Test.Day1.Demo_2(Day1.java:33)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Demo_2 -->
  <testcase name="Test_3" time="0.005" classname="Test.Day2"/>
  <testcase name="MobileLoginHomeLoan" time="0.004" classname="Test.Day4"/>
  <testcase name="MobileLoginCarLoan" time="0.004" classname="Test.Day3"/>
  <testcase name="MobileSignInCarLoan" time="0.002" classname="Test.Day3"/>
  <testcase name="WebLoginCarLoan" time="0.002" classname="Test.Day3"/>
</testsuite> <!-- Test -->
